multimodal/src/autogluon/multimodal/utils/model.py

```diff
@@ -223,6 +223,8 @@ def create_model(
             embedding_arch=model_config.embedding_arch,
             num_classes=num_classes,
             ffn_d_hidden=OmegaConf.select(model_config, "ffn_d_hidden", default=192),
+            additive_attention=model_config.additive_attention,
+            share_qv_weights=model_config.share_qv_weights,
```

Comment:
To be backward compatible, use
 `
additive_attention=OmegaConf.select(model_config, "additive_attention", default=False)
share_qv_weights=OmegaConf.select(model_config, "share_qv_weights", default=False)
`